run_witan_app.sh

```diff
@@ -6,6 +6,8 @@ cat <<EOF  > /root/.witan-app.edn
                      :keyspace "witan"
                      :replication 3}
  :s3 {:bucket "witan-${ENVIRONMENT?not defined}-data"}}
+:kafka {:host "master.mesos"
+         :port 9092}
```

Comment:
Ok, yes, rather than specify 'a broker' it should discover from ZK `(zk/brokers)` and use a broker list as comma-separated string: https://cwiki.apache.org/confluence/display/KAFKA/0.8.0+Producer+Example